ansible/group_vars/all.yml

```diff
@@ -40,6 +40,18 @@ insecure_registrys:
 # The port that the Kubernetes apiserver component listens on.
 kube_master_api_port: 443
 
+# Required for HA master configuration
+# Kubernetes master apiserver interface
+#kube_apiserver_interface: eth1
+
+# Required for HA master configuration
+# A floating ip address for multiple kube-apiservers
```

Comment:
I suggest changing floating to a virtual. Make note that this address should not be assigned to any hosts. The address should come from the same network used to assign real IP addresses to Kubernetes Masters.